rust-runtime/aws-smithy-types/src/error/unhandled.rs

```diff
@@ -58,6 +61,7 @@ impl Builder {
 /// [`DisplayErrorContext`](crate::error::display::DisplayErrorContext), use another
 /// error reporter library that visits the error's cause/source chain, or call
 /// [`Error::source`](std::error::Error::source) for more details about the underlying cause.
+#[deprecated(note = "This type is no longer used by errors.")]
```

Comment:
so is this type still used at all?

Reply:
It's no longer used. I left it in to be removed in the next release in case anyone was referencing it.